File: java/Tarea17.java
/*
Nombre: Jesus Martin Guajardo Olivo
Matricula: 1838264
Hora: N2
Descripcion: Generar numeros nones
*/
import javax.swing.JOptionPane;
public class Tarea22{
	public static void main(String[] args) {
		int i,j,n,m,valor,veces=0;
		String entrada,letrero="";
		int[][] arreglo;
		entrada = JOptionPane.showInputDialog("Introduce la cantidad de filas del arreglo");
		n = Integer.parseInt(entrada);
		arreglo = new int[n];
		for (i=0;i<n;i++) {
				entrada = JOptionPane.showInputDialog("Introduce la posicion "+(i+1)+","+(j+1));
				arreglo[i] = Integer.parseInt(entrada);
		}
		entrada = JOptionPane.showInputDialog("Cual valor buscas en la matriz");
		valor = Integer.parseInt(entrada);
		for (i=0;i<n;i++) {
				if(arreglo[i]==valor){
					veces++;
			}
		}
		JOptionPane.showMessageDialog(null,"el valor "+valor+" se encontro "+veces+" veces");
		for(i=0;i<n;i++){
			letrero = letrero + arreglo[i] + " ";
		}
		JOptionPane.showMessageDialog(null,letrero);
	}//Fin del metodo main
}//Fin de la clase

File: Ruby/Tarea17.rb
print("Introduce la cantidad de filas del arreglo")
n = gets().to_i
arreglo = Array.new(n)
for i in (0...n)
	print("Introduce la posicion #{i}")
	arreglo[i] = gets().to_i
end
print("Cual valor busca en el arreglo?")
valor = gets().to_i
for i in (0...n)
	if arreglo[i]==valor
		veces++
	end
end
print("El valor #{valor} se encontro #{veces} veces en el arreglo")
for i in (0...n)
	print("#{arreglo[i]}")
end